js/util/ajax.js

```diff
@@ -59,7 +61,11 @@ exports.getImage = function(url, callback) {
             (window.URL || window.webkitURL).revokeObjectURL(img.src);
         };
         var blob = new window.Blob([new Uint8Array(imgData)], { type: 'image/png' });
-        img.src = (window.URL || window.webkitURL).createObjectURL(blob);
+        if (imgData.byteLength) {
```

Comment:
Why did you decide to branch on `imgData.byteLength` rather than "status is 204 No Content"?